ROMFS/px4fmu_common/init.d/airframes/1103_standard_vtol_sih.hil

```diff
@@ -55,9 +53,9 @@ param set-default CA_SV_CS1_TYPE 3  # elevator
 param set-default CA_SV_CS2_TRQ_Y 1
 param set-default CA_SV_CS2_TYPE 4  # rudder
 
-param set-default FW_AIRSPD_MAX 12
-param set-default FW_AIRSPD_MIN 7
-param set-default FW_AIRSPD_TRIM 10
+param set-default FW_AIRSPD_MAX 20
+param set-default FW_AIRSPD_MIN 10
+param set-default FW_AIRSPD_TRIM 15
```

Comment:
Does it fly nice at this higher airspeed?

Reply:
With the params here, it only reaches about 12 m/s where it flies perfectly. When increasing SIH_T_MAX to 6.5N (corresponding to CA_ parameters? is CT = max thrust?) it can do 18 m/s which still looks good -- here's a log to judge w a second set of eyes: https://logs.px4.io/plot_app?log=bcb83c1e-2399-419f-9970-79389b84a3e6 (large airspeed spike was right when I increased SIH_T_MAX)

Reply:
also keep in mind that there is still this magic number doubling the forward thrust: 

https://github.com/mbjd/PX4-Autopilot/blob/5bca71791a27db30e89e8904c40cab51e11f8e8b/src/modules/simulation/simulator_sih/sih.cpp#L344

and probably more of this type of hackiness in sih.cpp. so the sims should be taken with several grains of salt until the models are overhauled and thoroughly validated.

Reply:
Okay but then I would be in favor or reverting these changes for the airspeed setpoint again, and address modelling in a separate work.

Reply:
reverted :+1: